test/integration/targets/pids/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/posix/group3
+skip/freebsd
+skip/osx
```

Comment:
Why are these platforms skipped? Is `pidof` not available as a package on these systems?

Reply:
Yes. They are not in that platforms.

Reply:
I just checked and `pidof` is available on macOS via `brew` and FreeBSD via `pkg`. Here's an example of how to install a package with `brew` in your test:

https://github.com/ansible/ansible/blob/f2f743aef039e4f7241b8d8e186baa11cbe625c9/test/integration/targets/archive/tasks/main.yml#L43-L60

So you should be able to include both those platforms in your test.